lib/manageiq/api/common/rbac/unshare_resource.rb

```diff
@@ -0,0 +1,30 @@
+require 'rbac-api-client'
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class UnshareResource < ShareResource
+          attr_accessor :count
```

Comment:
Minor but blank line after the attr_accessors as they are not a "part" of initialize